shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -589,6 +623,14 @@ void updateSemantics(ByteBuffer buffer, String[] strings) {
                     sendAccessibilityEvent(selectionEvent);
                 }
             }
+            if (object.hasFlag(Flag.IS_LIVE_REGION)) {
+                String priorLabelHint = mLiveRegions.get(object.id);
+                String labelHint = object.getLabelHint();
+                mLiveRegions.put(object.id, labelHint);
+                if (priorLabelHint == null) {
```

Comment:
What if the priorLabelHint was different? Should there be no announcement that it changed?